gcp/api/server.py

```diff
@@ -188,24 +191,24 @@ def build_determine_version_result(
     # this requirement.
     missed_empty_buckets = (inverted_empty_bucket_bitmap & bitmap).bit_count()
 
-    estimated_num_diff = estimate_diff(
-        _BUCKET_SIZE -
-        bucket_matches_by_proj[idx.key]  # Buckets that match are not changed
+    estimated_diff_files = estimate_diff(
+        _BUCKET_SIZE  # Starting with the total number of buckets
+        - bucket_matches_by_proj[idx.key]  # Buckets that match are not changed
         - empty_bucket_count  # Buckets that are empty are not changed
         + missed_empty_buckets  # Unless they don't match the bitmap
         - num_skipped_buckets,  # Buckets skipped are assumed unchanged
         abs(idx.file_count - max_files)  # The difference in file count
     )
 
     version_match = osv_service_v1_pb2.VersionMatch(
-        score=(max_files - estimated_num_diff) / max_files,
+        score=(max_files - estimated_diff_files) / max_files,
         minimum_file_matches=file_matches_by_proj[idx.key],
-        estimated_diff_files=estimated_num_diff,
+        estimated_diff_files=estimated_diff_files,
         repo_info=osv_service_v1_pb2.VersionRepositoryInformation(
             type=osv_service_v1_pb2.VersionRepositoryInformation.GIT,
             address=idx.repo_addr,
             commit=idx.commit,
-            version=idx.version,
+            version=idx.tag.removeprefix(_TAG_PREFIX),
```

Comment:
Is it confusing to have this field called `version` when it refers to a git tag? Is `RepoIndex.version` used elsewhere?

Reply:
You're right it could be confusing, the main reason to keep it version here is to have the API not change for this update, though since it's still in experimental we probably should change it now rather than later.

Reply:
Changed to tag now, turns out there's already a tag entry in the protobuf.